02: Reading from the disk/run.sh

```
@@ -0,0 +1 @@
+qemu-system-i386 -fda build/main_floppy.img
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)